pkgs/development/interpreters/gnudatalanguage/default.nix

```diff
@@ -0,0 +1,159 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+# Native build inputs for the compiler
+, cmake
+, enableGAppsWrapping ? true
+, wrapGAppsHook ? null
+# Build dependencies
+, readline
+, ncurses #? curses
+, zlib
+, gsl
+, openmp
+, graphicsmagick #? imagemagick
+, netcdf
+, hdf4
+, hdf5
+, fftw, fftwFloat, fftwLongDouble
+, proj
+, shapelib
+, expat
+, udunits
+, eigen
+, pslib # a C library for PostScript
+, eccodes
+, glpk
+, libpng
+, plplot
+, libtiff
+, libgeotiff
+, libjpeg # for hdf4 and hdf5
+, libtirpc
+, pythonInterpreter ? python3, python3 ? null
+, enableMPI ? (stdenv.isLinux || stdenv.isDarwin)
+, libraryMPI ? mpich #? openmpi
+, mpich ? null
+# Unfree optional dependency
+# for hdf4 and hdf5
+, enableSzip ? false
+, szip ? null
+, enableWX ? true
+, wxWidgets ? wxGTK31, wxGTK31 ? null
+# X11: OFF by default since wxWidgets SHOULD take preference in ALL builds
+# https://github.com/gnudatalanguage/gdl/blob/master/CMakeLists.txt#L437
+, enableXWin ? false, xorg ? null
```

Comment:
```suggestion
, wxWidgets ? wxGTK31, wxGTK31
# X11: OFF by default since wxWidgets SHOULD take preference in ALL builds
# https://github.com/gnudatalanguage/gdl/blob/master/CMakeLists.txt#L437
, enableXWin ? false, xorg
```